dhis-2/dhis-test-integration/src/test/resources/tracker/te_with_tea_metadata.json

```diff
@@ -90,6 +91,7 @@
       "translations": [],
       "attributeValues": [],
       "legendSets": [],
+      "preferredSearchOperator": "IEQ",
```

Comment:
The I variant operators should not be allowed to be preferredSearchOperator. Since this is a new feature, we will not be breaking anything. So let us block this in this PR.